projects/plugins/jetpack/modules/sso/jetpack-sso-admin-create-user.css

```
@@ -0,0 +1,9 @@
+
+.jetpack-sso-admin-create-user-invite-message {
+	width: 550px;
+}
+/* Select the second-to-last tr in the table */
+#createuser .form-table tr:nth-last-child(2) {
+	display: none;
+}
```

I would do this and change the code comment, currently the comment just complies the code to the reader, it should explain the reasoning. 

```suggestion
/* Hide the checkbox to send WP core invitation emails when SSO is on */
#createuser .form-table tr:has( #send_user_notification )  {
	display: none;
}
```